### DISCUSSION TITLE
CustomTube Beta is discontinued (for now)

### DISCUSSION THREAD

**Discussion by lightbeam24** [lightbeam24/CustomTube#80] [Announcements]
Thanks to everyone who used CustomTube Beta and helped report bugs. Going forward, CustomTube will be getting smaller, yet more frequent updates, as opposed to huge updates that release once every couple months. So the next update will be 1.2.1, not 1.3.0. This renders Betas largely unnecessary, and even inconvenient.

This means that anyone currently using CustomTube Beta should switch to the stable version of CustomTube, which can be downloaded from addons.mozilla.org and the Chrome Web Store. Don't forget to disable/delete the Beta version of the extension, as it will cause issues if run alongside the regular version. Sorry if this is an inconvenience to anyone, but with the more frequent release schedule, you would have to download a new Beta every few days, and that would get annoying.

There may be a point in the future where a Beta release may become necessary again, but for now, this is how it's going to be.

**Tl;dr: If you use CustomTube Beta, switch to the regular version.**
